Public/Src/FrontEnd/MsBuild/Tracing/Log.cs

```diff
@@ -226,20 +226,30 @@ internal Logger()
             (ushort)LogEventId.ProjectIsNotSpecifyingTheProjectReferenceProtocol,
             EventGenerators = EventGenerators.LocalOnly,
             EventLevel = Level.Warning,
-            Message = Events.LabeledProvenancePrefix + "Project '{projectName}' is not specifying its project reference protocol, and therefore the targets to call cannot be inferred. " +
+            Message = Events.LabeledProvenancePrefix + "Project '{projectName}' is not specifying its project reference protocol, and therefore the targets to call on its dependencies cannot be inferred. " +
                       "Falling back to calling the project default targets. For more details, see https://github.com/Microsoft/msbuild/blob/master/documentation/specs/static-graph.md",
             EventTask = (ushort)Events.Tasks.Engine,
             EventOpcode = (byte)Events.Tasks.Parser,
             Keywords = (int)(Events.Keywords.UserMessage | Events.Keywords.Diagnostics))]
         public abstract void ProjectIsNotSpecifyingTheProjectReferenceProtocol(LoggingContext context, Location location, string projectName);
 
+        [GeneratedEvent(
+            (ushort)LogEventId.ProjectPredictedTargetsAlsoContainDefaultTargets,
+            EventGenerators = EventGenerators.LocalOnly,
+            EventLevel = Level.Warning,
+            Message = Events.LabeledProvenancePrefix + "Default targets were appended to the predicted target of project '{projectName}'. " +

```

Comment:
>Default targets [](start = 56, length = 15)

Should list the default targets added in this message (DRI reduction)

Reply:
Yeap, good point. Added.

---
In reply to: [283011368](https://github.com/microsoft/BuildXL/pull/293#discussion_r283011368) [](ancestors = 283011368)